### PR TITLE
Update/gstreamer tutorial

### PR DESCRIPTION
**Pull request by xinyuli1204**
1. update gstreamer tutorial

### CODE REVIEW

File: docs/tutorials/gstreamer.mdx [hunk 1]
```diff
@@ -1 +1,100 @@
...
+# Build GStreamer pipeline with opencv-python 
+
+This tutorial demonstrates how to use OpenCV with GStreamer to capture video from cameras. It supports different camera configurations, pixel formats, and frame size.
+
+## Tutorial
+
+### Prerequisites
+1. opencv-python with gstreamer backend
+3. **windows**:  
+    Install the C++ version of sensing-dev. make sure GST_PLUGIN_PATH is set.
```
Comment (Fixstars-momoko) [inline docs/tutorials/gstreamer.mdx:14]: can you mention `-InstallGstPlugins` option here?

File: docs/tutorials/gstreamer.mdx [hunk 2]
```diff
@@ -1 +1,100 @@
...
+3. **linux**:
```
Comment (Fixstars-momoko) [inline docs/tutorials/gstreamer.mdx:15]: can you mention `--install-gst-tools` here too?

File: docs/tutorials/gstreamer.mdx [hunk 3]
```diff
@@ -1 +1,100 @@
...
+    Install the C++ version of sensing-dev. make sure GST_PLUGIN_PATH is set.
+3. **linux**:  
+   Set the GST_PLUGIN_PATH environment variable and sensing-dev library_path:
+   ```bash
+   export GST_PLUGIN_PATH=/opt/sensing-dev/lib/x86_64-linux-gnu/gstreamer-1.0
+   export LD_LIBRARY_PATH=/opt/sensing-dev/lib:/opt/sensing-dev/lib/x86_64-linux-gnu:$LD_LIBRARY_PATH
+   ```
+
+### Configuration
+Please adjust the camera_name, pixelformat, width, height, framerate based on your requirements
```
Comment (Fixstars-momoko) [inline docs/tutorials/gstreamer.mdx:23]: can you mention that you can obtain these values are obtained by arv-tool-0.8? Reference: https://github.com/Sensing-Dev/doc/blob/027fc72487ddedc54bc2a74cfaf279f68973919f/docs/lessons/camera.md?plain=1#L26

File: docs/tutorials/gstreamer.mdx [hunk 4]
```diff
@@ -1 +1,100 @@
+---
+sidebar_position: 4
+---
 
+# Build GStreamer pipeline with opencv-python 
+
+This tutorial demonstrates how to use OpenCV with GStreamer to capture video from cameras. It supports different camera configurations, pixel formats, and frame size.
+
```
Comment (Fixstars-momoko) [inline docs/tutorials/gstreamer.mdx:8]: Can you shortly introduce appsink and opencv-python with gstreamer here? Like "while you can build and run pipeline with gstreamer API, this element and opencv api combination allows you to get an output of gstreamer pipeline" or something like that.

Comment (xinyuli1204) [reply docs/tutorials/gstreamer.mdx:9]: done

File: docs/tutorials/gstreamer.mdx (hunk 2 above)
Comment (xinyuli1204) [reply docs/tutorials/gstreamer.mdx:16]: done

File: docs/tutorials/gstreamer.mdx (hunk 4 above)
Comment (Fixstars-momoko) [reply docs/tutorials/gstreamer.mdx:10]: confirmed

**Review by Fixstars-momoko — APPROVED** [review]

**Pull request merged**